lib/backend/k8s/resources/workloadendpoint.go

```diff
@@ -47,11 +52,29 @@ func (c *WorkloadEndpointClient) Create(ctx context.Context, kvp *model.KVPair)
 }
 
 func (c *WorkloadEndpointClient) Update(ctx context.Context, kvp *model.KVPair) (*model.KVPair, error) {
-	log.Warn("Operation Update is not supported on WorkloadEndpoint type")
-	return nil, cerrors.ErrorOperationNotSupported{
-		Identifier: kvp.Key,
-		Operation:  "Create",
+	log.Debug("Received Update request on WorkloadEndpoint type")
```

Comment:
I wasn't sure where to this patching of pod IP should end up. `Update` made the most sense to me and I've put it here. Let me know if this needs to be somewhere else.

Reply:
I think Update is the right place.